File: libwebrtc-sys/src/logging.cc
#include "libwebrtc-sys/include/logging.h"

void set_arcas_log_to_stderr(bool value)
{
    rtc::LogMessage::SetLogToStderr(value);
    RTC_LOG(LS_INFO) << "Set log to stderr";
}

void set_arcas_log_level(rtc::LoggingSeverity level)
{
    rtc::LogMessage::LogToDebug(level);
    RTC_LOG(LS_INFO) << "Logging level set to " << level;
}

File: libwebrtc-sys/src/logging.rs
#[cxx::bridge]
pub mod ffi {
    unsafe extern "C++" {
        include!("include/logging.h");

        #[namespace = "rtc"]
        type LoggingSeverity = crate::shared_bridge::ffi::LoggingSeverity;

        // Logging
        fn set_arcas_log_to_stderr(log: bool);
        fn set_arcas_log_level(level: LoggingSeverity);
    }
}
